tests/lib/rules/prefer-ember-test-helpers.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require('../../../lib/rules/prefer-ember-test-helpers');
+const RuleTester = require('eslint').RuleTester;
+
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: 'module',
+  },
+});
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const TEST_FILE_NAME = 'some-test.js';
+
+ruleTester.run('prefer-ember-test-helpers', rule, {
+  valid: [
+    {
+      filename: TEST_FILE_NAME,
+      code: `import { blur } from '@ember/test-helpers';
+
+      test('foo', async (assert) => {
+        await blur();
+      });`,
+    },
+
+    {
+      filename: TEST_FILE_NAME,
+      code: `import { find } from '@ember/test-helpers';
+
+      test('foo', async (assert) => {
+        await find();
+      });`,
+    },
+
+    {
+      filename: TEST_FILE_NAME,
+      code: `import { focus } from '@ember/test-helpers';
+
+      test('foo', async (assert) => {
+        await focus();
+      });`,
+    },
+  ],
```

Comment:
Let's add a valid test case to ensure we can handle import aliasing:
```
import { focus as myFocusName } from 'irrelevant-import-path';
...
myFocusName()
```

Reply:
👍

Reply:
I had a typo in this one, just fixed it.